packages/blocks/src/api/test/registration.js

```diff
@@ -1705,42 +1697,6 @@ describe( 'blocks', () => {
 				'Block bindings source "core/test-source" is already registered.'
 			);
 		} );
-
-		it( 'should correctly merge properties when bootstrap happens after registration', () => {
```

Comment:
I wonder if this should be allowed or not now that server sources get bootstrapped sooner in the process.

Reply:
Now that we force the server to register the source sooner than the client, only the client could override the server, not the other way around.